Microsoft.Toolkit.Mvvm/Microsoft.Toolkit.Mvvm.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <LangVersion>8.0</LangVersion>
     <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
This is what keeps it encapsulated to just the library and not developers using this, eh?

Reply:
Yup, same as with the base `Microsoft.Toolkit` package, it just tells the C# compiler to allow `unsafe` blocks 🙂